src/styles/fontFamily/index.native.js

```diff
@@ -0,0 +1,16 @@
+import bold from './bold';
+
+const fontFamily = {
+    EXP_NEUE_ITALIC: 'ExpensifyNeue-Italic',
+    EXP_NEUE_BOLD: bold,
```

Comment:
I'm not sure, that's how it was before so I kept the same behaviour.

Reply:
@rushatgabhane currently the extra fonts are applied only to web except for bold, for bold they are applied to all platforms except Android. If you want me to change it I could do one of two things.

### Option 1

Add the emoji font families in `index.js` instead of `bold/index.js`.

```js
fontFamily = {
    ...
    EXP_NEUE_BOLD: bold + ', Segoe UI Emoji, Noto Color Emoji',
}
```

### Option 2

Add a `bold/index.ios.js` without the emoji fonts.